conda-forge.yml

```diff
@@ -2,4 +2,6 @@ channel_priority: strict
 azure:
   settings_win:
     pool:
-      vmImage: windows-2019
\ No newline at end of file
+      vmImage: windows-2019
+os_version:
+  linux_64: cos7
```

Comment:
could you revert this file before we ping for review

Reply:
@hmaarrfk Done! Thanks!